### PR TITLE
Vit-servicing-station v0.yaml API doc fixes

### PR DESCRIPTION
**Pull request by Mr-Leshiy**
Add fixes for the API doc corresponding to the https://editor.swagger.io/

### CODE REVIEW

File: src/vit-servicing-station/doc/api/v0.yaml [hunk 1]
```diff
@@ -207,7 +207,7 @@ paths:
 
   /api/v0/challenges/{id}/{voter_group_id}:
     get:
-      operationId: getChallenge
+      operationId: getChallengeByIdAndVoterGroupId
       summary: Get challenge by id
```
Comment (minikin) [inline src/vit-servicing-station/doc/api/v0.yaml:211]: Should we update the summary to `Get challenge by id and voter group id`?

File: src/vit-servicing-station/doc/api/v0.yaml [hunk 2]
```diff
@@ -428,13 +428,10 @@ paths:
...
             schema:
-              Proposals:
-                 properties:
-                  type: array
-                  description: list of Proposals entries in json format
-                  required: true
-                  items:
-                    $ref: "#/components/schemas/Proposal"
+              type: array
+              description: list of Proposals entries in json format
```
Comment (minikin) [inline src/vit-servicing-station/doc/api/v0.yaml:432]: Let's capitalize the `List`word.

**Review by minikin — APPROVED** [review]
Approved, with minor comments. Thank you!

File: src/vit-servicing-station/doc/api/v0.yaml (hunk 1 above)
Comment (Mr-Leshiy) [reply src/vit-servicing-station/doc/api/v0.yaml:211]: yes, sure, good catch

**Pull request merged**